code/modules/mob/living/carbon/human/human.dm

```diff
@@ -334,10 +344,22 @@
 		dat += "<BR><b>Left hand:</b> <A href='?src=\ref[src];item=[slot_l_hand]'>[istype(l_hand) ? l_hand : "nothing"]</A>"
 		dat += "<BR><b>Right hand:</b> <A href='?src=\ref[src];item=[slot_r_hand]'>[istype(r_hand) ? r_hand : "nothing"]</A>"
 
-	// Do they get an option to set internals?
-	if(istype(wear_mask, /obj/item/clothing/mask) || istype(head, /obj/item/clothing/head/helmet/space))
-		if(istype(back, /obj/item/tank) || istype(belt, /obj/item/tank) || istype(s_store, /obj/item/tank))
-			dat += "<BR><A href='?src=\ref[src];item=internals'>Toggle internals.</A>"
+	var/has_mask // 0, no mask | 1, mask but it's down | 2, mask and it's ready
+	var/has_helmet
+	if(istype(wear_mask, /obj/item/clothing/mask))
+		var/obj/item/clothing/mask/M = wear_mask
+		has_mask = 1
+		if(!M.hanging)
+			has_mask = 2
+	if(istype(head, /obj/item/clothing/head/helmet/space))
```

Comment:
Doesn't this include softsuits, which can't hold atmosphere on their own? I know voidsuits can, is there a way to check for that?

Reply:
From what I can see, softsuit helmets can hold atmosphere on their own

Reply:
i checked and this seems to be true